cluster/Vagrantfile

```diff
@@ -102,7 +102,11 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
       elsif node_os == UBUNTU
         c.vm.box = 'ubuntu/xenial64'
       else
-        c.vm.box = 'centos/7'
+        if ENV['CONTIV_KUBEADM']
+           c.vm.box = 'contiv/kubeadm-centos'
```

Comment:
@rchirakk: How was this box built? Do we have some scripts/documentation on how to build this VM?